src/commands/deploy/nextjs.ts

```diff
@@ -224,7 +228,8 @@ async function deployCDN(
             origin: serverOrigin,
         },
     );
-    if (!distributionUrl.startsWith("https://") && !distributionUrl.startsWith("http://")) {
+
+    if (!distributionUrl.startsWith("https://")) {
```

Comment:
Why this change? What if the `distributionUrl` ever gets to be [http://genezio.com](http://genezio.com)? This will result into the link being `https://http://genezio.com`